data/database.py

```diff
@@ -1414,6 +1418,19 @@ class Meta:
         )
 
 
+class UploadedBlob(BaseModel):
```

Comment:
Is this new table to avoid adding an extra new column to the BlobUpload table?

Reply:
And because they have different lookup semantics; we could reuse BlobUpload but we'd have to verify the content checksum is in there

Reply:
@kurtismullins have a better name? :) I couldn't come up with one, although I didn't spend a lot of time on it.

Reply:
haha, great question! What would you consider the primary difference between BlobUpload and UploadedBlob? That might lead into some sort of way to differentiate the names.

Reply:
I can't come up with a better name either, so I am going to resolve this thread.